lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -159,6 +167,10 @@ class AwsCompileKafkaEvents {
                 prefix = 'security_group:';
                 needsEc2Permissions = true;
                 break;
+              case 'saslPlainAuth':
+                type = 'BASIC_AUTH';
```

Comment:
I tried the change directly since the BASIC_AUTH matched what was displayed in the AWS Management Console and it worked perfectly.

![image](https://user-images.githubusercontent.com/2861984/123949973-1c20c500-d99b-11eb-95f3-b3a4bc930fba.png)

Reply:
I did indeed managed to deploy the following function with this change:

```yaml
functions:
  productConsumer:
    handler: src/handlers/product-consumer.handler
    events:
      - kafka:
          accessConfigurations:
            saslPlainAuth: arn:aws:secretsmanager:eu-west-1:1234567890:secret:/mysecret
          topic: ${self:custom.kafka.productUpdateTopic}
          bootstrapServers:
            - abc3.xyz.com:9092
            - abc2.xyz.com:9092
```

Reply:
Sounds great :clap: Did you also test out that the Lambda is properly invoked? Sorry for asking so many questions, but I just want to make sure that it's working correctly before merging and I currently don't have a good environment to try to replicate and confirm it.

Reply:
Sorry I missed this. You are completely right to be cautious. 
I've created a test serveless deployment and subscribed to a Confluent Cloud cluster that only support SASL/PLAIN and it worked straight away! @pgrzesik Let me know if we want to organise a screensharing session.

Reply:
![image](https://user-images.githubusercontent.com/2861984/124095229-61073300-da51-11eb-97ec-4babba86df33.png)

![image](https://user-images.githubusercontent.com/2861984/124095461-9a3fa300-da51-11eb-8c58-8e67ae1affa9.png)

![image](https://user-images.githubusercontent.com/2861984/124095796-eee31e00-da51-11eb-9e3e-e6c04053e023.png)

Reply:
Perfect, thanks a lot for confirming :bow: One thing before merging - could you please rebase on top of current master and push again? We had a slight change in CI jobs to include Node 16 in our testing matrix and I'd like to make sure that this PR runs tests against newest config. Thanks in advance :bow:

Reply:
👍  Done.

Reply:
@overbit hey! congratulations on the endeavor! May I ask you how is your confluent cloud cluster setup? Does it work for basic/standard (low cost dev env) or only for dedicated ones?

Reply:
Unfortunately this feature has not been released yet.  Once a new serverless framework version is released you would be able  to use it with any ConfluentCloud instance type (I tested it with a basic/standard cluster and it worked flawlessly).

@pgrzesik  I'm assuming that a new release is imminent since the [serverless docs](https://www.serverless.com/framework/docs/providers/aws/events/kafka#simple-event-definition) already shows this feature and it can be confusing for users

Reply:
Hello @overbit - the release will definitely happen in a day or two - sorry for the confusion